src/platforms/esp32/main/platform_defaultatoms.h

```diff
@@ -179,6 +186,13 @@
 #define STA_CONNECTED_ATOM TERM_FROM_ATOM_INDEX(STA_CONNECTED_ATOM_INDEX)
 #define STA_DISCONNECTED_ATOM TERM_FROM_ATOM_INDEX(STA_DISCONNECTED_ATOM_INDEX)
 #define STA_DHCP_HOSTNAME_ATOM TERM_FROM_ATOM_INDEX(STA_DHCP_HOSTNAME_INDEX)
+#define AP_ATOM TERM_FROM_ATOM_INDEX(AP_ATOM_INDEX)
+#define AP_SSID_HIDDEN_ATOM TERM_FROM_ATOM_INDEX(AP_SSID_HIDDEN_ATOM_INDEX)
+#define AP_MAX_CONNECTIONS_ATOM TERM_FROM_ATOM_INDEX(AP_MAX_CONNECTIONS_ATOM_INDEX)
```

Comment:
same here, atom is `max_connections` so the constant should be called `MAX_CONNECTIONS_ATOM`

Reply:
Fixed